packages/auth/package.json

```diff
@@ -6,34 +6,32 @@
   "bugs": "https://github.com/heroku/cli/issues",
   "dependencies": {
     "@heroku-cli/color": "^1.1.14",
-    "@heroku-cli/command": "^8.5.0",
-    "@oclif/command": "^1.5.11",
-    "@oclif/config": "^1.12.10",
+    "@heroku-cli/command": "^9.0.0",
+    "@oclif/core": "^1.21.0",
     "cli-ux": "^4.9.3",
     "date-fns": "^2.0.0-alpha.8"
   },
   "devDependencies": {
-    "@fancy-test/nock": "^0.1.1",
     "@heroku-cli/dev-cli": "^1.26.13",
     "@heroku-cli/schema": "^1.0.25",
-    "@oclif/plugin-help": "^2.1.6",
-    "@oclif/test": "^1.2.4",
+    "@oclif/test": "^2.2.13",
     "@types/chai": "^4.1.7",
     "@types/mocha": "^5.2.6",
     "@types/nock": "^9.3.1",
     "@types/node": "^10.12.24",
+    "@oclif/plugin-help": "^2.2.0",
     "@types/supports-color": "^5.3.0",
     "chai": "^4.2.0",
     "eslint": "^6.7.2",
     "eslint-config-oclif": "^3.1.0",
     "eslint-config-oclif-typescript": "^0.1.0",
-    "globby": "^9.0.0",
+    "globby": "^10.0.1",
     "mocha": "^5.2.0",
     "nock": "^10.0.6",
     "nyc": "^13.2.0",
     "ts-node": "^10.9.1",
     "tslib": "^1.9.3",
-    "typescript": "3.7.5"
+    "typescript": "4.8.4"
   },
   "engines": {
     "node": ">=10.0.0"
```

Comment:
Might as well update this to >=12.0.0 (oclif/core doesn't support earlier versions of Node)